tests/in_order_test.py

```diff
@@ -0,0 +1,151 @@
+import pytest
+
+from mockito import (mock, when, VerificationError,
+                     verifyZeroInteractions, ensureNoUnverifiedInteractions)
+from mockito.inorder import InOrder
+from mockito.mock_registry import mock_registry
+
+
+def test_in_order_should_observe_single_mock():
+    m = mock()
+    in_order: InOrder = InOrder([m])
+    assert m in in_order.mocks
```

Comment:
in_order.mocks is an implementation detail we don't expose and test.  I still think that this is true.